homeassistant/components/media_player/sonos.py

```diff
@@ -608,6 +613,13 @@ def update(self):
             else:
                 playlist_size = int(playlist_size)
 
+            if playlist_size is None or playlist_size == 0:
```

Comment:
I'll give it a go :)

btw, just noticed that the macOS Sonos Controller app also disables the play/pause button upon an empty playlist (apparently the iOS app doesn't, that's where I checked earlier)